### ISSUE TITLE
Exception when resetting an annotation description

### ISSUE THREAD

**Issue by petebankhead** [qupath/qupath#661]
## Bug report

**Describe the bug**
In v0.2 it isn't possible to set an annotation description, then reset it (by removing the text).

**To Reproduce**
1. Create an annotation
2. Change the description in the properties and apply the change
3. Delete the description and (try to) apply the change

Alternatively use this script:

```groovy
def roi = ROIs.createRectangleROI(0, 0, 100, 100, ImagePlane.getDefaultPlane())
def annotation = PathObjects.createAnnotationObject(roi)
annotation.setDescription("something")
annotation.setDescription(null)
```

**Expected behavior**
Setting description to null or an empty string removes it.

**Desktop (please complete the following information):**
 - QuPath Version 0.2

**Additional context**
See https://forum.image.sc/t/unable-to-clear-annotation-description-in-qupath/47741

Should be quite straightforward to fix, but subtleties take a bit of thought.

**Comment by yli-hallila** [comment]
I've fixed this bug on my branch like this: https://github.com/yli-hallila/qupath/blob/master/qupath-core/src/main/java/qupath/lib/objects/PathAnnotationObject.java#L77-L81

**Comment by petebankhead** [comment]
Closing this as the fix has been merged and will be included in v0.3.
